### DISCUSSION TITLE
Can it be used on seperate Xamarin Android/iOS apps?

### DISCUSSION THREAD

**Discussion by fdhsdrdark** [xamarin/XamarinCommunityToolkit#929] [Q&A] [answered]
Newbie question for the Community Toolkit package. 

Can it be used on seperate Xamarin Android/iOS apps, or it's only aiming Xamarin Forms projects?

If not, is there any chance we see android/iOS versions of the package in the future?

Thanks very much

**Comment by jfversluis** [comment, marked as answer]
Sorry! This is for Xamarin.Forms only and we don't have any plans to change that.

If you do need anything you can of course look at the Android/iOS specific code and use that in your project. But this project is very specifically targeting Forms

**Reply by fdhsdrdark** [reply]
OK, thank you very much!
